caluma/form/serializers.py

```diff
@@ -390,7 +390,34 @@ class Meta:
         model = models.Option
 
 
+def create_and_link_child_documents(form, document):
```

Comment:
Nitpick - I'd probably move this to a method within `DocumentSerializer`, as it's not used anywhere else. Unless you expect this to happen in the near future. But then, it may need a more "general" location

Reply:
Good point! Fixed.